### PR TITLE
Ryan trinh development week 3 feature 2

### PR DESCRIPTION
**Pull request by rqtrinh**
Display contact list on the page after the clicking get contacts button

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -12,21 +12,26 @@
 import com.google.cloud.datastore.QueryResults;
 import com.google.cloud.datastore.StructuredQuery.OrderBy;
 import com.google.gson.Gson;
+
+import org.jsoup.Jsoup;
```
Comment (RyanSCollins) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:21]: Keep sorted.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -38,3 +38,15 @@ async function showHello() {
     const helloContainer = document.getElementById('hello-container');
     helloContainer.innerText = msg;
 }
+
+async function showContacts() {
+    const responseFromServer = await fetch('/form-handler');
+    const textFromResponse = await responseFromServer.json();
+
+    String = Jsoup.clean(request.getParameter("title"), Whitelist.none());
+    //Pick a random message
```
Comment (RyanSCollins) [inline portfolio/src/main/webapp/script.js]: This section needs to be updated.

**Review by RyanSCollins — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java (hunk 1 above)
Comment (rqtrinh) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:21]: I resorted the imports.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (rqtrinh) [reply portfolio/src/main/webapp/script.js]: Updated this section in the latest commit.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 3]
```diff
@@ -1,18 +1,27 @@
 package com.google.sps.servlets;
 
+import java.util.*;
```
Comment (RyanSCollins) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: java.util.ArrayList

**Review by RyanSCollins — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 3 above)
Comment (rqtrinh) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Changed in latest push

**Review by RyanSCollins — APPROVED** [review]

**Pull request merged**